README.md

```diff
@@ -50,7 +50,7 @@ See [this document](https://github.com/hardkoded/puppeteer-sharp/blob/master/CON
 ## Take screenshots
 
 <!-- snippet: ScreenshotAsync -->
-<a id='snippet-ScreenshotAsync'></a>
+<a id='snippet-screenshotasync'></a>
```

Comment:
I need to fix this. I bet you are in MacOS? I'm getting the same changes but the bot on github actions changes these back.

Reply:
I'm on Windows and would really like if these could be fixed, I'm reverting them over and over